assets/Gruntfile.js

```diff
@@ -207,6 +208,8 @@ module.exports = function (grunt) {
           'bower_components/uri.js/src/URI.min.js',
           'bower_components/uri.js/src/jquery.URI.min.js',
           'bower_components/uri.js/src/URI.fragmentQuery.js',
+          // bower registration error? we get 2x versions of uri.js/urijs
+          'bower_components/urijs/',
```

Comment:
I'd like to understand why we have two versions. Is this only happening with your changes?